snapcraft/plugins/nodejs.py

```diff
@@ -64,6 +63,9 @@ def schema(cls):
             },
             'default': [],
         }
+        schema['properties']['node-engine'] = {
+            'type': 'string'
```

Comment:
Yes, but this breaks backwards compatibility so we now are stuck with it. We update snapcraft often enough that we can always keep it using the best choice available